test/guest/network-cockpit.xml

```diff
@@ -1,4 +1,4 @@
-<network>
+<network ipv6='yes'>
```

Comment:
do we have mechanics in place on the test runners to recreate the network if the definition changed?